src/test/scala/BuildersSpec.scala

```diff
@@ -32,7 +32,7 @@ class BuildersSpec extends FunSpec {
           .withHead(new BodyHeadBuilder()
             .withHeadline("News Article")
             .withByline("It took a lot of work to get there")
-            .withAbstract(<p>It wasn't easy, but they <em>never</em> gave up!</p>)
+            .withAbstract(new AbstractBuilder().withTextParagraph("It wasn't easy, but they never gave up!"))
```

Comment:
I suppose you had to remove the `em` to avoid having to parse the input string. Any chance the input will ever contain tags and/or references?

Reply:
I removed the `<em>` because I removed/deprecated the methods that accept XML.

The input can still contain an `em`, except we'd expect it to be added, type-safely, like this:
```scala
.withAbstract(new AbstractBuilder()
  .withParagraph(new ParagraphBuilder()
    .withText("It wasn't easy, but they ")
    .withEmphasis(Em(Seq(dataRecord("never"))))
    .withText(" gave up!")
  )
)
```
In another project, I intend to create a utility that converts Jsoup nodes to NITF nodes so that we can parse HTML and convert it to NITF.